src/Parallel/GlobalCache.hpp

```diff
@@ -333,6 +336,9 @@ class GlobalCache : public CBase_GlobalCache<Metavariables> {
   template <typename GlobalCacheTag, typename Function, typename... Args>
   void mutate(const std::tuple<Args...>& args) noexcept;
 
+  /// Retrieve the proxy to the global cache
+  self_proxy_type get_self_proxy() noexcept;
```

Comment:
[optional] would it be better to name this `get_this_proxy` to follow Charm++ conventions?